test/joy/DeployTest.php

```diff
@@ -17,7 +17,7 @@ public function testDeploy()
     {
         $this->init(self::RECIPE);
         $this->tester->run([
-            'deploy',
+            'deploy:invoke',
```

Comment:
Better to create separate test for it. As normal deploy test is also needed. Or even better - unit test for this part of the code.